go.mod

```diff
@@ -93,6 +93,7 @@ require (
 	github.com/spf13/cobra v1.4.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/ssor/bom v0.0.0-20170718123548-6386211fdfcf // indirect
+	github.com/stretchr/objx v0.4.0 // indirect
```

Comment:
Wonder where that is coming from 🤔

Reply:
This is from me using testify's mock package